src/components/Explore/TokenRow.tsx

```diff
@@ -297,8 +342,15 @@ export default function TokenRow({
       </FavoriteCell>
       <ListNumberCell>{listNumber}</ListNumberCell>
       <NameCell>
-        <CurrencyLogo currency={currency} />
-        {tokenName} <TokenSymbol>{tokenSymbol}</TokenSymbol>
+        <DesktopView>
+          <CurrencyLogo currency={currency} size={'24px'} />
```

Comment:
delete this Desktop and Mobile --> just make this 24 px always for now and figure out later